### PR TITLE
Flash: Use ChannelUpdates for fee calculations

### PR DESCRIPTION
**Pull request by omerfirmak**

### CODE REVIEW

File: source/agora/common/Amount.d [hunk 1]
```diff
@@ -285,6 +285,22 @@ public struct Amount
         return this.value / other.value;
     }
 
+    /***************************************************************************
+
+        Params:
```
Comment (AndrejMitrovic) [inline source/agora/common/Amount.d:293]: Could use some documentation about what the function actually does.

File: source/agora/flash/Config.d [hunk 2]
```diff
@@ -129,3 +129,30 @@ unittest
...
 }
+
+/***************************************************************************
+
+    Calculate total fee from the given update and payment total
+
+    Params:
+        update = latest update for the channel
+        total = payment total
+
```
Comment (AndrejMitrovic) [inline source/agora/flash/Config.d:140]: Missing `Returns`

File: source/agora/common/Amount.d (hunk 1 above)
Comment (omerfirmak) [reply source/agora/common/Amount.d:293]: Oops

**Comment by omerfirmak** [comment]
Updated

**Review by AndrejMitrovic — APPROVED** [review]

**Pull request merged**